docs/notebooks/demo/gretel-demo-sequence-of-events.ipynb

```diff
@@ -0,0 +1,456 @@
+    "    project_name=\"gretel-demo-events\",\n",
+    "    api_key=\"prompt\",\n",
+    "    cache=\"yes\",\n",
+    "    validate=True,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define base DGAN configuration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create a custom dgan config to modify some base setting\n",
+    "import yaml\n",
+    "from gretel_client.gretel.config_setup import create_model_config_from_base\n",
+    "\n",
+    "config = create_model_config_from_base(\n",
+    "    \"time-series\",\n",
+    "    params={\n",
+    "        \"apply_example_scaling\": True,\n",
+    "        \"max_sequence_len\": MAX_SEQUENCE_LEN,\n",
+    "        \"sample_len\": SAMPLE_LEN,\n",
+    "    },\n",
+    "    example_id_column=EXAMPLE_COLUMN,\n",
+    ")\n",
+    "\n",
+    "with open(\"custom_base_dgan_config.yaml\", \"w\") as file:\n",
+    "    yaml.dump(config, file, default_flow_style=False, sort_keys=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Train DGAN models leveraging Gretel Tuner"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from utils import EventTypeHistogramAndTransitionDistance\n",
+    "\n",
+    "# This cell should take ~10 minutes to complete.\n",
+    "tuner_config = \"\"\"\n",
+    "base_config: custom_base_dgan_config.yaml\n",
+    "\n",
+    "params:\n",
+    "\n",
+    "    attribute_loss_coef:\n",
+    "        choices: [1, 5, 10]\n",
+    "\n",
+    "    attribute_num_layers:\n",
+    "        choices: [3, 4]\n",
+    "\n",
+    "    attribute_num_units:\n",
+    "        choices: [50, 100, 200]\n",
+    "\n",
+    "    batch_size:\n",
+    "        choices: [100, 200]\n",
+    "\n",
+    "    epochs:\n",
+    "        choices: [1000, 2000, 4000, 8000]\n",
+    "\n",
+    "    generator_learning_rate:\n",
+    "        log_range: [0.000001, 0.001]\n",
+    "\n",
+    "    discriminator_learning_rate:\n",
+    "        log_range: [0.000001, 0.001]\n",
+    "\n",
+    "\"\"\"\n",
+    "\n",
+    "target_job = \"tune-dgan\"\n",
+    "\n",
+    "metric = EventTypeHistogramAndTransitionDistance(reference_df=data_source, example_id_column=EXAMPLE_COLUMN, event_column=EVENT_COLUMN, num_samples=5000)\n",
+    "\n",
+    "tuner_results = gretel.run_tuner(\n",
+    "    tuner_config,\n",
+    "    data_source=data_source,\n",
+    "    n_jobs=4,\n",
+    "    n_trials=16,\n",
+    "    metric=metric,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "\n",
+    "# show best config \n",
+    "best_config = tuner_results.best_config\n",
+    "yaml.dump(config, sys.stdout, default_flow_style=False, sort_keys=False)"
```

Comment:
Should you be showing `best_config` rather than `config`?

Also, I think this is simpler and will work

```python 
print(yaml.dump(best_config))
```

Then you can remove the `sys` import.